### ISSUE TITLE
Did you managed to get didFinishLaunchingWithOptions called within this delegate?

### ISSUE THREAD

**Issue by sinosoidal** [colede/qt-app-delegate#1]
Hi,

Did you managed to get didFinishLaunchingWithOptions called within this delegate?

Regards,

Nuno

**Comment by colede** [comment]
Yes. It the QtAppDelegate.mm file you can see i wrote a NSLog comment so
you can check to see that it loads.

**Comment by sinosoidal** [comment]
Humm... strange...

Some of the calls are being called but this on in specific not.

Do you have any ideas of some details that is really important to pay attention to?

**Comment by colede** [comment]
It's been a while since I looked at that. I have a more full fledged
project that I was working on on a different system. If I get a chance I
will check it out to see if there is something I missed when I stripped it
down for that sample project. Either way, I think the general idea of how
to do it is illustrated if you use that as a research point. I figured it
all out from scratch mostly.

**Comment by sinosoidal** [comment]
I will have a look again. 

Please let me know if you have any tip that might be escaping me. 

Thanks for sharing on a sample project!

**Comment by colede** [comment]
You're welcome. if you find anything wrong and make a change feel free to
push to the project

**Comment by sinosoidal** [comment]
Hi again,

I have run your project on a device, thing that I haven't done yet and didFinishLaunchingWithOptions isn't called. Can you please checkout the version you have on github and confirm if it works on your device? Maybe you didn't commit your last changes. 

Thanks,

Regards,

Nuno

**Comment by colede** [comment]
I don't have a device presently, but I was testing on an iphone4 with ios7
I believe.

**Comment by toby20130333** [comment]
Hi all
I meet the same problem about didFinishLaunchingWithOptions which isn't called by system

**Comment by toby20130333** [comment]
hi anybody exist?

**Comment by pixelecho** [comment]
for me didFinishLaunchingWithOptions is also not called...... but dont have a solution.

**Comment by wschacker** [comment]
didFinishLaunchingWithOptions s also not called. do you know what's solution

**Comment by sinosoidal** [comment]
Do this:

@interface QIOSApplicationDelegate
@end
//! Add a category to QIOSApplicationDelegate
@interface QIOSApplicationDelegate (QFacebookApplicationDelegate)
@end
@implementation QIOSApplicationDelegate (QFacebookApplicationDelegate)
- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions
  {
  // do stuff here
  
  return YES;
  }
- (BOOL)application:(UIApplication *)application openURL:(NSURL *)url sourceApplication:(NSString *)sourceApplication annotation:(id)annotation
  {
  return [[FBSDKApplicationDelegate sharedInstance] application:application openURL:url sourceApplication:sourceApplication annotation:annotation];
  }
  @end

Nuno Santos
Founder / CEO / CTO
www.imaginando.pt
+351 91 621 69 62
